### PR TITLE
Release 1.11.0

### PR DESCRIPTION
**Pull request by rowanmanning**
I'm working on this release in a PR so we can review it as a whole. Please don't merge through GitHub – I'll do it locally when it's all reviewed.

I've reviewed both @Rayraegah's and @joshski's work, but someone might want to look through my commits and check them out – they're mostly documentation changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,29 +24,39 @@ Setup
...
 cp config/development.sample.json config/development.json
 cp config/production.sample.json config/production.json
 cp config/test.sample.json config/test.json
 ```
 
-Each of these files defines configurations for a different environment. If you're just running the application locally, then you should be OK with just development configurations. The [available configurations are documented here](#configurations).
+Each of these files defines configurations for a different environment. If you're just running the application locally, then you should be OK with just development and test configurations. The [available configurations are documented here](#configurations).
 
-Now that you've got your application configured, make sure you have a server running with the `mongod` command in another terminal window. You can then run in each mode with the following commands:
+Now that you've got your application configured, make sure you have a MongoDB server running with the `mongod` command in another terminal window. you can run in each mode by changing the `NODE_ENV` environment variable:
```
Comment (andrewmee) [inline README.md]: Should be uppercase "You"

File: README.md [hunk 2]
```diff
@@ -23,29 +24,39 @@ Setup
...
 
 ```sh
-NODE_ENV=production node index.js   # Run in production
-NODE_ENV=development node index.js  # Run in development
-NODE_ENV=test node index.js         # Run in test
+NODE_ENV=development node index.js
```
Comment (andrewmee) [inline README.md:56]: Was this an intentional deletion?  It seems clearer to me with the variations given as examples rather than just one.

**Comment by andrewmee** [comment]
I've reviewed your commits and all look good apart from the couple of issues noted above.

File: README.md (hunk 2 above)
Comment (rowanmanning) [inline README.md:56]: Yep intentional, and to match the pa11y-webservice docs. Does the line above not explain it well enough?

> You can run in each mode by changing the `NODE_ENV` environment variable:

File: README.md [hunk 3]
```diff
@@ -23,29 +24,39 @@ Setup
 
 pa11y-dashboard requires [Node.js][node] 0.10+ and [PhantomJS][phantom]. See the [pa11y][pa11y] documentation for detailed instructions on how to install these dependencies on your operating system.
 
-You'll also need to have [MongoDB][mongo] installed and running. See the [MongoDB install guide][mongo-install] for more information on this. 
+You'll also need to have [MongoDB][mongo] installed and running. See the [MongoDB install guide][mongo-install] for more information on this.
 
-You'll then need to clone this repo locally and install dependencies with `npm install`.
+You'll then need to clone this repo locally and install dependencies with `npm install`. Now we need to add some configuration before we can run the application. We can do this in two ways:
```
Comment (dotcode) [inline README.md]: Can we stick with "you", rather than switching to "we"? Do you think that would read a little better?

Comment (rowanmanning) [inline README.md]: Yep, either way should be consistent!

**Pull request merged**